app/controllers/admin/banners_controller.rb

```diff
@@ -0,0 +1,47 @@
+module Admin
+  class BannersController < BaseController
+    layout 'admin'
+
+    def index
+      @banners = Banner.where('expires_at > ?', DateTime.now)
```

Comment:
Will need an index on expires_at. Might also be a good idea to automatically delete expired banners when they are accessed (if there's no way to see them) even for admins.

Reply:
Done, added an index on expires_at. And I added a `before_filter :delete_expired_banners, only: [:index]` as I can't think of any other way to do it automatically without creating a scheduled rake task.